tests/test_multiline_variables.py

```diff
@@ -0,0 +1,22 @@
+from universum.lib.module_arguments import ModuleArgumentParser, multiline_argument
+
+
+def test_multiline_vars(tmp_path, monkeypatch):
+    text = "This is text\nwith some line breaks\n"
+    parser = ModuleArgumentParser()
+    parser.add_argument("--argument", type=multiline_argument)
+
+    # Usual value
+    namespace = parser.parse_args(["--argument", text])
+    assert namespace.argument == text
+
+    # File
+    tmp_file = tmp_path / "myfile.txt"
+    tmp_file.write_text(text)
+    namespace = parser.parse_args(["--argument", '@' + str(tmp_file)])
+    assert namespace.argument == text
+
+    # Stdin
+    monkeypatch.setattr('sys.stdin.read', lambda: text)
```

Comment:
As an alternative, you can spawn separate interpreter

Reply:
I tried subprocess & multiprocessing, but liked this one more for better readability.